packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -26,21 +26,30 @@ export class AtomicSegmentedFacetScrollable {
   }
 
   private renderArrow(arrowDirection: string) {
-    return (
+    return [
```

Comment:
I prefer using `<Fragment>` instead of `[`, but we do use `[` most often.

Reply:
End result is the same in html